tms_objects2.erb

```diff
@@ -442,6 +446,7 @@ objects_main_query = tmsclient.execute("SELECT * FROM [Coboat_ObjectsMainQuery]
 #586 - ExhVenuesAltName |an| - nameActorSet - appellationValue - AltVenueName - NULL
 #ExhVenuesAltName |an| - nameActorSet - appellationValue - NameType - (No NULL -> NameType)
 #if statement for ExhVenuesAltName |an| - nameActorSet - appellationValue - always occurs
+#5011 - vitalDateActor - latestDate - conEdate - 1999
```

Comment:
just say:
#5011 - vitalDateActor - earliesDatet & latestDate

as just saying latestDate with one of the values is a little hard to follow.